OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -153,15 +156,19 @@ Actor ChooseTarget(Actor self, WRange range)
 
 			return inRange
 				.Where(a =>
+				{
+					var target = Target.FromActor(a);
```

Comment:
Above I meant for `TargetedActor` to be a target, because we want that to become invalidated if it is captured (or teleported, or a few other conditions).  After a second look, that field is actually unused, which is a bit odd.

You don't need to worry about the target here, because it won't be invalidated in the course of this query.